libs/components/help-inline/src/lib/modules/help-inline/help-inline.component.ts

```diff
@@ -117,27 +107,40 @@ export class SkyHelpInlineComponent {
   @Output()
   public actionClick = new EventEmitter<void>();
 
-  protected isPopoverOpened: boolean | undefined;
-  protected popoverId: string | undefined;
-  protected popoverTemplate: TemplateRef<unknown> | undefined;
+  protected readonly defaultAriaLabel: Signal<string | undefined>;
+  protected readonly labelTextResolved: Signal<string | undefined>;
 
-  protected readonly helpSvc = inject(SkyHelpService, { optional: true });
+  readonly #format = inject(SkyAppFormat);
+  readonly #labelText = signal<string | undefined>(undefined);
+  readonly #resourcesSvc = inject(SkyLibResourcesService);
 
-  #_popoverContent: string | TemplateRef<unknown> | undefined;
+  constructor() {
+    this.defaultAriaLabel = toSignal(
+      this.#resourcesSvc.getString('skyux_help_inline_button_title'),
+    );
 
-  readonly #idSvc = inject(SkyIdService);
+    const labelTextTemplate = toSignal(
+      this.#resourcesSvc.getString('skyux_help_inline_aria_label'),
+    );
 
-  protected onClick(): void {
-    this.actionClick.emit();
+    this.labelTextResolved = computed<string | undefined>(
+      (): string | undefined => {
+        const labelText = this.#labelText();
+
+        if (labelText) {
+          const resource = labelTextTemplate();
 
-    if (this.helpKey) {
-      this.helpSvc?.openHelp({
-        helpKey: this.helpKey,
-      });
-    }
+          if (resource) {
+            return this.#format.formatText(resource, this.#labelText());
+          }
+        }
+
+        return;
+      },
```

Comment:
☝️ Is there a better way to handle a signaled resources string that needs formatting?

Reply:
I would convert `#labelText` to an Observable, `switchMap` it with the call to the resources service, then convert back to a Signal.

```
  protected labelTextResolved = toSignal(
    toObservable(this.#labelText).pipe(
      switchMap((labelText) =>
        this.#resourcesSvc.getString('skyux_help_inline_aria_label', labelText),
      ),
    ),
  );
```